src/SQLite.cs

```diff
@@ -3138,9 +3552,22 @@ public IEnumerable<T> ExecuteDeferredQuery<T> (TableMapping map)
 				else if (map.Method == TableMapping.MapMethod.ByName) {
 					MethodInfo getSetter = null;
 					if (typeof(T) != map.MappedType) {
-						getSetter = typeof(FastColumnSetter)
-							.GetMethod (nameof(FastColumnSetter.GetFastSetter),
-								BindingFlags.NonPublic | BindingFlags.Static).MakeGenericMethod (map.MappedType);
+#if NET8_0_OR_GREATER
+						// The runtime feature switch must be on a separate 'if' branch on its own,
+						// or the analyzer might not be able to correctly follow the program flow.
+						if (!RuntimeFeature.IsDynamicCodeSupported) {
+							if (map.MappedType.IsValueType) {
+								getSetter = null;
+							}
+							else {
+								getSetter = FastColumnSetter.GetFastSetterMethodInfoUnsafe (map.MappedType);
+							}
+						}
+						else {
+							getSetter = FastColumnSetter.GetFastSetterMethodInfoUnsafe (map.MappedType);
+						}
+#endif
+						getSetter = FastColumnSetter.GetFastSetterMethodInfoUnsafe (map.MappedType);
```

Comment:
Is this line supposed to be there? Or is it supposed to be in an `#else` branch?

Reply:
Oops, typo, fixed in https://github.com/praeclarum/sqlite-net/pull/1222/commits/4eacdc21f4d534c219b80f75c8083218c2825037! Thank you 🙂